doc/release-notes.md

```diff
@@ -66,9 +66,21 @@ platform.
 Notable changes
 ===============
 
-Example item
+Updated RPCs
 ------------
 
+Note: some low-level RPC changes mainly useful for testing are described in the
+Low-level Changes section below.
+
+* The `sendmany` RPC had an argument `minconf` that was not well specified and
+  would lead to RPC errors even when the wallet's coin selection would succeed.
+  The `sendtoaddress` RPC never had this check, so to normalize the behavior,
```

Comment:
The reasoning here is flawed, BTW. `sendtoaddress` was kind of soft-deprecated by `sendmany`.

Reply:
It is just one of many reasons why the arg should be ignored. Feel free to remove that reasoning, the others should still hold.